CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ Categories: Added, Removed, Changed, Fixed, Nonfunctional, Deprecated
 
 ## Unreleased
 
+## 3.2.0 (2022.09.20)
+
+### Added
+ - [Resolve #1225] Added Python 3.10 support (#1227)
+ - Implemented a `sceptre dump config` command (#1220)
+ - [Resolve #1224] Add --drifted option to `drift show` command (#1246)
+ - [Resolve #1212] Conditional Stacks via "ignore" and "obsolete";
+   `sceptre prune` command for cleaning up obsolete stacks (#1229)
+
+### Changed
+ - [Resolve #1225] Updating `troposphere` version for python 3.10 compatibility (#1226)
```

Comment:
this might cause incompatability for troposhere users but probably does not warrant a major version update.

Reply:
The troposphere version update only affects (a) dev requirements and (b) the extras version requirement. I don't think it should be breaking at all.

Reply:
Oh, I really don't think so. We use a lot of Jinja stuff directly. It's pretty core to Sceptre.